scripts/build-corpus.py

```
@@ -191,6 +224,10 @@ def pct(a, b):
       + " · ".join(f"`{s}` {c}" for s, c in scope_counts.most_common()) + ".")
     w("- **Gerund league table:** "
       + " · ".join(f"{g} {c}" for g, c in gerunds.most_common(24)) + ".")
+    w(f"- **Phrase length** (words between `smoother` and the stamp, gerund "
```

Ruff (0.16.3)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)